Coinbase Pro.lua

```diff
@@ -123,10 +130,15 @@ function queryCoinbaseProApi(endpoint)
         return JSON(content):dictionary()
 end
 
-function queryExchangeRates(currency)
-        local url = string.format("https://api.pro.coinbase.com/products/%s-%s/ticker", currency, nativeCurrency)
-        local content = Connection():request("GET", url)
-        return JSON(content):dictionary()
+function queryExchangeRate(product_id)
+        return queryCoinbaseProApi("products/" .. product_id .. "/ticker")["price"]
```

Comment:
This actually intentionally did not use queryCoinbaseProApi because the ticker requests don't need to be signed.

Reply:
Even though they are public, I think using this should not be detrimental at all. At least it looks cleaner that way :D